src/charm.py

```diff
@@ -17,23 +18,19 @@
 class FastAPIDemoCharm(ops.CharmBase):
     """Charm the service."""
 
-    def __init__(self, framework):
+    def __init__(self, framework: ops.Framework) -> None:
         super().__init__(framework)
         self.pebble_service_name = "fastapi-service"
-        self.container = self.unit.get_container(
-            "demo-server"
-        )  # see 'containers' in charmcraft.yaml
-
-        # Charm events defined in the database requires charm library.
+        self.container = self.unit.get_container("demo-server")  # see 'containers' in charmcraft.yaml
```

Comment:
I'm pretty sure this line is too long now, which is why it was wrapped before.

(When we move the tests to the early chapters we should also make sure that linting is also part of that).